src/models/managed-serial-port.ts

```diff
@@ -21,6 +21,7 @@ export type ManagedSerialPort = {
     status: Status;
     subscriptions: string[];
     subscribedTo: string[];
+    readState: ReadState,
```

Comment:
`readState` war doch erforderlich. Habe es im letzten Commit entfernt

Reply:
Aber kriegst du wirklich vom Tauri ein ReadState? Ich dachte readState ist er dann da, wenn der Port offen ist. 🤔

Reply:
ka das wurde automatisch generiert wie auch der commit

Reply:
Ich habe noch mal geguckt. ReadState kann ruhig weg. Ich habe die dafür benötigte Änderungen in `src\components\serial-port\SerialPort.vue` gemacht. Du kannst `src\components\serial-port\SerialPort.vue` und src\models\managed-serial-port.ts einfach von `main` übernehmen